### PR TITLE
add json and code of edx.forum.comment.created

### PR DESCRIPTION
**Pull request by tehreem-sadat**
**Trello Link:** _https://trello.com/c/aRdDOubS/12-discussion-forum-event-edxforumcommentcreated_

**Description:** _Users create a comment about a response by entering text and then submitting the contributions. When these actions are complete, the server emits an edx.forum.comment.created event._

**Checks before merge:**

- [x] Reviewed
- [ ] Code Reviewed
- [x] Unit Test Pass
- [x] Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/tests/current/edx.forum.comment.created.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    "id": "5bc88be5977195008e000004",
+    "options": {
+      "followed": false
+    },
+    "response": {
+      "id": "5bc889c5977195008e000003"
+    },
+    "truncated": false,
+    "url": "http://edx.devstack.lms:18000/courses/course-v1:edX+DemoX+Demo_Course/discussion/forum/i4x-edx-eiorguegnru-course-foobarbaz/threads/5bc887e1977195008e000000",
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/tests/current/edx.forum.comment.created.json]: Please replace all occurrences of `edx.devstack.lms` with `localhost`.

Comment (tehreem-sadat) [reply openedx/features/caliper_tracking/tests/current/edx.forum.comment.created.json]: done

File: openedx/features/caliper_tracking/transformers/forum_transformers.py [hunk 2]
```diff
@@ -39,3 +39,46 @@ def edx_forum_response_created(current_event, caliper_event):
...
+    :param current_event: default log
+    :param caliper_event: log containing both basic and default attribute
+    :return: final created log
+    """
+
+    current_event_details = current_event['event']
+    current_event_details.pop('url', '')
+
+    caliper_object = {
+        'body': current_event_details.pop('body', ''),
```
Comment (asamolion) [inline openedx/features/caliper_tracking/transformers/forum_transformers.py]: Why is there a need to specify a default value for body?

Comment (tehreem-sadat) [reply openedx/features/caliper_tracking/transformers/forum_transformers.py]: @asamolion  This case will never run, It is just for precaution. Should i remove it ?

Comment (tehreem-sadat) [reply openedx/features/caliper_tracking/transformers/forum_transformers.py]: Done (Y)

**Pull request merged**